Merchant/App/Settings/SettingsViewController.swift

```diff
@@ -14,18 +14,18 @@ class SettingsViewController: BDCViewController {
     
     fileprivate let cellId = "settingsCell"
     
-    var tableView: UITableView = {
+    let tableView: UITableView = {
         let tableView = UITableView()
         tableView.translatesAutoresizingMaskIntoConstraints = false
         tableView.separatorStyle = .none
         return tableView
     }()
-    var companyNameTextField = BDCTextField.build(.type1)
-    var destinationAddressTextField = BDCTextField.build(.type1)
-    var pinCodeLabel = BDCLabel.build(.subtitle)
-    var selectedCurrencyLabel = BDCLabel.build(.subtitle)
-    var currenciesPickerView = UIPickerView(frame: .zero)
-    var currenciesView: UIView = {
+    let companyNameTextField = BDCTextField.build(.type1)
+    let destinationAddressTextField = BDCTextField.build(.type1)
+    let pinCodeLabel = BDCLabel.build(.subtitle)
+    let selectedCurrencyLabel = BDCLabel.build(.subtitle)
```

Comment:
fileprivate please

Reply:
In this commit was the objective to correct the value type only. Talking about the access control I would leave this for another commit which will do only that as there are possible many more places than just these...